client-sdk/ts-web/rt/playground/webpack.config.js

```diff
@@ -1,10 +1,24 @@
-module.exports = {
-    mode: 'development',
-    output: {
-        library: {
-            name: 'playground',
-            type: 'window',
-            export: 'playground',
+module.exports = [
+    {
+        mode: 'development',
+        output: {
+            library: {
+                name: 'playground',
+                type: 'window',
+                export: 'playground',
+            },
         },
     },
-};
+    {
+        mode: 'development',
+        entry: './src/consensus.js',
+        output: {
+            library: {
+                name: 'playground-consensus',
+                type: 'window',
+                export: 'playground-consensus',
```

Comment:
:one: does this work? it's still exported as `playground` and accessed from cypress as `playground`

Reply:
Good spot, i never tested this with cypress locally but always manually via browser. Updated so that `playground` is exported for both and now it works correctly with cypress.

The main problem was that before the cypress spec passed if the `window.playground` object was undefined. I updated the spec so that it now also ensures that the `window.playground` is actually defined.